src/styles/blocks/gallery.scss

```diff
@@ -0,0 +1,89 @@
+.gallery {
+  display: flex;
+  margin: 0 auto;
+  column-gap: 0;
```

Comment:
column-gap should have a value other than 0 for mobile view to maintain spacing between columns when flex is used.